src/components/AppNavigation/Settings.vue

```diff
@@ -139,6 +153,30 @@ export default {
 		settingsTitle() {
 			return this.$t('calendar', 'Settings & import').replace(/&amp;/g, '&')
 		},
+		slotDurationOptions() {
+			return [{
+				label: this.$t('calendar', '5 minutes'),
```

Comment:
MomentJS should be able to provide these translations out of the box, right?

Reply:
excellent point!